app/controllers/lifehacks_controller.rb

```diff
@@ -0,0 +1,7 @@
+class LifehacksController <  ApplicationController
```

Comment:
Unnecessary spacing detected.<br>Operator `<` should be surrounded with a single space.